ci-operator/jobs/openshift-priv/cluster-baremetal-operator/openshift-priv-cluster-baremetal-operator-release-4.7-presubmits.yaml

```diff
@@ -178,7 +178,7 @@ presubmits:
       skip_cloning: true
     hidden: true
     labels:
-      ci-operator.openshift.io/cloud: packet
+      ci-operator.openshift.io/cloud: packet-edge
```

Comment:
@smg247 After running `make update`, here has been changed to `packet-edge` from `packet`, not sure why.

Reply:
Never mind, I addressed it by rebasing master branch.